http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -313,29 +313,25 @@ object KubernetesServiceDbQueries {
                               labelFilter: LabelMap
   ) = {
     val query = for {
-      (((((((((app, nodepool), cluster), label), clusterNamespaces), appNamespace), services), disks), diskLabels),
-       appError
-      ) <- baseAppQuery join
+      ((((((((app, nodepool), cluster), label)), services), disks), diskLabels), appError) <- baseAppQuery join
         baseNodepoolQuery on (_.nodepoolId === _.id) join
         baseClusterQuery on (_._2.clusterId === _.id) joinLeft
         labelQuery on { case (c, lbl) =>
           val resourceTypeFilter = lbl.resourceId
             .mapTo[AppId] === c._1._1.id && lbl.resourceType === LabelResourceType.app
           resourceTypeFilter
         } joinLeft
-        namespaceQuery on (_._1._2.id === _.clusterId) join
-        namespaceQuery on (_._1._1._1._1.namespaceId === _.id) joinLeft
-        serviceQuery on (_._1._1._1._1._1.id === _.appId) joinLeft
-        persistentDiskQuery.tableQuery on (_._1._1._1._1._1._1.diskId === _.id) joinLeft
+        serviceQuery on (_._1._1._1.id === _.appId) joinLeft
+        persistentDiskQuery.tableQuery on (_._1._1._1._1.diskId === _.id) joinLeft
         labelQuery on { case (c, lbl) =>
           lbl.resourceId
-            .mapTo[DiskId] === c._1._1._1._1._1._1._1.diskId && lbl.resourceType === LabelResourceType.persistentDisk
+            .mapTo[DiskId] === c._1._1._1._1._1.diskId && lbl.resourceType === LabelResourceType.persistentDisk
         } joinLeft
-        appErrorQuery on (_._1._1._1._1._1._1._1._1.id === _.appId)
-    } yield (cluster, nodepool, app, label, clusterNamespaces, appNamespace, services, disks, diskLabels, appError)
```

Comment:
This used to distinguish between `clusterNamespaces` and `appNamespace` - what has changed, now that only APP has namespaceName? Are there any consequences?

Reply:
In the original implementation, `clusterNamespaces` is all the namespaces for a given cluster, and `appNamespace` is the namespace a given app is in. I removed `namespaces` from `KubernetesCluster`, so we don't need to look up and aggregate all namespaces for a given cluster anymore, since I don't think we need it

Reply:
ok - does the new code respect the weird uniqueness requirements? "the Galaxy installation requires that release names be unique within a cluster", from LeoAppServiceInterp?

Reply:
Ok, got it from the description; it can't make that guarantee.

If a collision does occur, is there a log or failure case that would alert us to the issue? What would happen?

Reply:
Do we have to drop `namespaces` from `KubernetesCluster`? We could keep it in just to maintain the uniqueness constraint

Reply:
Do we use `KubernetesCluster.namespaces`? If not I'd vote to remove it (as this PR does). 

In general I think it's better to model uniqueness at the DB level, instead of through code. What if we just added a DB unique constraint on `APP.release` and `APP.namespace`?

Reply:
That works for me, my understanding was that the uniqueness needed to be enforced by cluster, not across the whole app table, but this would fit the bill for sure

Reply:
Yeah, technically uniqueness just needs to be per cluster. But that's hard to model in our current DB schema. We use UUIDs anyway so it should be fine (and collisions will fail fast if enforced at the DB level).

Thinking ahead, if we drop `KUBERNETES_CLUSTER` and `NODEPOOL` tables in the future, and move `cloudContext` to the `APP` table, we could make the unique constraint on `(namespace, cloudContext)`.

Reply:
>In general I think it's better to model uniqueness at the DB level, instead of through code. What if we just added a DB unique constraint on APP.release and APP.namespace?

So I checked DB and there are apps use the same `release` and `namespace`...I think these are galaxy apps. When we reuse disk for galaxy app, we seem to use the same `release` and `namespace`. We can maybe add `namespace`, `destroyedDate` constraint in the next PR.

I think we can leave these constraints out of this PR for now just so the migration is a bit safer. And once this PR is on dev/prod, we can add appropriate constrains in the follow up PR (we still need to drop `namespace`table once things look good anyway).